.make-lint-translation-keys-expect

```diff
@@ -1,19 +1,35 @@
-resources/authgear/templates/en/web/authflowv2/select_account.html:13:24: invalid translation key: "app.name"
+resources/authgear/templates/en/web/authflowv2/__base_page_frame.html:43:12: translation key not defined: "page-frame-content"
+resources/authgear/templates/en/web/authflowv2/__bot_protection_dialog.html:4:25: translation key not defined: "dialog-controller-str"
+resources/authgear/templates/en/web/authflowv2/__bot_protection_dialog.html:5:19: translation key not defined: "dialog-content"
+resources/authgear/templates/en/web/authflowv2/__csrf_error_page_layout.html:28:18: translation key not defined: "page-content"
+resources/authgear/templates/en/web/authflowv2/__dialog.html:9:14: translation key not defined: "dialog-attr"
 resources/authgear/templates/en/web/authflowv2/__error.html:134:18: translation key not defined: "authflowv2/__error_account"
 resources/authgear/templates/en/web/authflowv2/__error.html:166:20: translation key not defined: "authflowv2/__error_account"
-resources/authgear/templates/en/web/authflowv2/setup_totp.html:34:20: invalid translation key: "copy-button-label"
-resources/authgear/templates/en/web/authflowv2/view_recovery_code.html:55:20: invalid translation key: "copy-button-label"
+resources/authgear/templates/en/web/authflowv2/__error.html:201:31: translation key not defined: "customer-support-link"
+resources/authgear/templates/en/web/authflowv2/__html_head.html:4:19: invalid translation key: "app.name"
+resources/authgear/templates/en/web/authflowv2/__page_frame.html:12:18: translation key not defined: "page-content"
+resources/authgear/templates/en/web/authflowv2/__settings_item.html:4:13: translation key not defined: "some-key"
+resources/authgear/templates/en/web/authflowv2/__settings_page_frame.html:46:18: translation key not defined: "page-content"
+resources/authgear/templates/en/web/authflowv2/__toc_pp_footer.html:23:54: invalid translation key: "terms-of-service-link"
+resources/authgear/templates/en/web/authflowv2/__toc_pp_footer.html:24:53: invalid translation key: "privacy-policy-link"
+resources/authgear/templates/en/web/authflowv2/account_linking.html:94:21: invalid translation key: "printf "v2.component.oauth-branding.default.label-%s" .ProviderType"
+resources/authgear/templates/en/web/authflowv2/enter_password.html:35:11: invalid translation key: "$description_key"
```

Comment:
Do not seem expected.

Reply:
We do not support `evaluating variables` in linter? ref https://github.com/authgear/authgear-server/blob/main/resources/authgear/templates/en/web/authflowv2/enter_password.html#L35

Reply:
1. Edit enter_password.html
  - Delete `"v2-enter-password-page-description"`
  - Delete `"v2-enter-additional-password-page-description"`
  - Eliminate `$description_key`
  - Keep L26-L32 and the `<p>`